src/orocos_cpp.cpp

```diff
@@ -125,9 +125,11 @@ bool OrocosCpp::initialize(const OrocosCppConfig& config, bool quiet)
     //Init Bundle
     if(config.init_bundle){
         if(!quiet) std::cout << "\nInitializing Bundle.." << std::endl;
+        bundle.reset(new Bundle);
         st = bundle->initialize(config.load_task_configs);
         if(!st){
             std::cerr << "Error during initialization of Bundle" << std::endl;
+            bundle.reset();
```

Comment:
Apparently there is no other way to ask if there is an active bundle then checking this public member object.

Reply:
If the variable should be public or not is one thing, but that it need to be initialized is right for sure.

Also it is right that is should happen here. The OrocosCPP class is thought of to be the single point of initialization (for simplification). If bundles are supposed to be used, they are initialized here.